ext/pdo/pdo_dbh.stub.php

```diff
@@ -1,7 +1,10 @@
 <?php
 
+/** @generate-function-entries */
+
 class PDO
 {
+    /** @alias PDO::dbh_constructor */
```

Comment:
I've just realized that internal (fake) methods can also be referenced as an alias